vsintegration/src/FSharp.Editor/Navigation/GoToDefinitionService.fs

```diff
@@ -30,74 +45,99 @@ type internal FSharpNavigableItem(document: Document, textSpan: TextSpan) =
         member this.DisplayTaggedParts = ImmutableArray<TaggedText>.Empty
         member this.ChildItems = ImmutableArray<INavigableItem>.Empty
 
+
 [<Shared>]
 [<ExportLanguageService(typeof<IGoToDefinitionService>, FSharpCommonConstants.FSharpLanguageName)>]
-type internal FSharpGoToDefinitionService 
-    [<ImportingConstructor>]
-    (
-        checkerProvider: FSharpCheckerProvider,
-        projectInfoManager: ProjectInfoManager,
-        [<ImportMany>]presenters: IEnumerable<INavigableItemsPresenter>
+type internal FSharpGoToDefinitionService [<ImportingConstructor>]
+    (   checkerProvider: FSharpCheckerProvider
+    ,   projectInfoManager: ProjectInfoManager
+    ,   signatureMetadataService : NavigateToSignatureMetadataService
+    ,   [<ImportMany>]_presenters: IEnumerable<INavigableItemsPresenter>
```

Comment:
most of the time i end up writing this way because i'm commenting and switching out services in the importing constructor, trying to figure out which one will actually do the right thing. it's more practical than stylistic